Added new kb article setting-font-size-accordion-header-net-maui (#826)
telerik/maui-docs: knowledge-base/setting-font-size-accordion-header-net-maui.md

---

> title: Setting Font Size for the text inside the Accordion Header

Reviewer: ```suggestion
title: Setting Font Size for the Text inside the Accordion Header
```

description: Learn how to set the font size for the text inside the AccordionItemHeader in the Accordion for .NET MAUI.
type: how-to
page_title: Setting Font Size for AccordionItemHeader in Accordion for .NET MAUI
slug: setting-font-size-accordionitemheader-net-maui
tags: accordion, accordionitemheader, font size, .net maui
res_type: kb
---

## Environment

| Version | Product | Author | 
| --- | --- | ---- | 
| 6.8.0 | Telerik UI for .NET MAUI Accordion| [Dobrinka Yordanova](https://www.telerik.com/blogs/author/dobrinka-yordanova)| 

## Description

> I want to set the font size for the text inside the AccordionItemHeader in the Accordion for .NET MAUI.

Reviewer: ```suggestion
How can I set the font size for the text inside the `AccordionItemHeader` in the Accordion for .NET MAUI?
```
